### PR TITLE
msg-chnl branch

### PR DESCRIPTION
**Pull request by BHess2653**

### CODE REVIEW

File: bot/controllers/channels.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+module.exports = () => {
+  const util = require('apex-util');
+
+  const _run = (message) => {
+    const ctrls = [
+      {
+        cmd: '!chnl',
```
Comment (LukeBillington) [inline bot/controllers/channels.js]: I suggest using a full command name like !channel or something like !announce or !broadcast, more for consistency with other commands within this bot, and to convey the message as what the command does in one word.

File: bot/controllers/channels.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+          util.log('Multiple Channels Parsing', chnls, 4);
+
+          chnls.map((chnl) => {
+            const targetChnl = message.guild.channels.find('name', chnl);
+            util.log('Asking API for Channel', targetChnl, 4);
+
+            if (targetChnl === null) {
+              return '"' + chnl + '" is not a known channel. Check text channels on the left to see what channels are available (They are case-sensitive)';
+            }
+            const msgs = msg.parsed;
```
Comment (LukeBillington) [inline bot/controllers/channels.js:28]: This prints out all the arguments passed into the string, including the command and selected channels. I suggest iterating over each of the arguments you really need. e.g: msg.parsed[2] and going forward and concatenate them with a space.

**Review by LukeBillington — CHANGES_REQUESTED** [review]
Hello there, great start, you got the looping through channels down.
• I'd try to find a better way to actually print out the message, without commas or the message arguments.
• You may want to consider renaming your command to something more semantic like !broadcast or !announce.
• Also I suggest adding the command to both the !help command in the index.js controller, and to the usage.md page.

File: bot/controllers/channels.js (hunk 1 above)
Comment (BHess2653) [reply bot/controllers/channels.js]: ok I made this change.

**Pull request closed without merging**